upup/models/cloudup/resources/addons/networking.kuberouter/k8s-1.12.yaml.template

```diff
@@ -1,4 +1,4 @@
-# Pulled and modified from https://github.com/cloudnativelabs/kube-router/blob/v0.3.1/daemonset/generic-kuberouter-all-features.yaml
+# Pulled and modified from https://github.com/cloudnativelabs/kube-router/blob/v0.4.0/daemonset/generic-kuberouter-all-features.yaml
 
 ---
```

Comment:
can we remove this `---` so that the comment above doesnt get treated as a separate yaml manifest? :) I remember us removing that from all the other addons as well, even though kops can now correctly handle that situation it'd be good to not have it in the `update cluster` diffs.

Reply:
No objection. Done.